LICENSE-Paulov-t.md

```diff
@@ -0,0 +1,26 @@
+# Paulov-t UNLICENSED
+
+Copyright (c) 2021-2024 [Paulov-t](https://github.com/paulov-t). All rights reserved.
+
+THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+**Paulov-t work is unlicensed**
+
+Any work or fork that is not Stay in Tarkov and uses any line of code from [Paulov-t](https://github.com/paulov-t), including but not limited to previous commits and forks of old commits is strictly prohibited. If you are using any [Paulov-t](https://github.com/paulov-t) code in your works, whether currently or previously, you can request permission directly from [Paulov-t](https://github.com/paulov-t) via GitHub, Email or Discord.
+
+**Redistribution**
+- You cannot reproduce and distribute copies of the Work (by [Paulov-t](https://github.com/paulov-t)) or Derivative Works (by [Paulov-t](https://github.com/paulov-t)) thereof in any medium, with or without modifications, and in Source or Object form.
```

Comment:
The license permits me to distribute in binary form and make modifications to the binary, I assume this is unintended.